packages/office-ui-fabric-react/src/components/ThemeGenerator/ThemeGenerator.ts

```diff
@@ -80,23 +80,22 @@ export class ThemeGenerator {
 
   /*
    * Gets code-formatted load theme blob that can be copy and pasted.
+   * Only used for the old theme designer, where loadTheme usage is acceptable,
+   * unlike in the new theme designer.
    */
   public static getThemeAsCode(slotRules: IThemeRules): any {
-    const attributeTemplate = "    {0}: '{1}',\n";
-    let output = '';
-
-    output += 'loadTheme({\n  palette: {\n';
+    const output = 'loadTheme({\n  palette: {\n';
+    return ThemeGenerator._makeRemainingCode(output, slotRules);
+  }
 
-    for (const ruleName in slotRules) {
-      if (slotRules.hasOwnProperty(ruleName)) {
-        const rule: IThemeSlotRule = slotRules[ruleName];
-        const camelCasedName = rule.name.charAt(0).toLowerCase() + rule.name.slice(1);
-        const outputColor = rule.color ? '#' + rule.color.hex : rule.value || '';
-        output += format(attributeTemplate, camelCasedName, outputColor);
-      }
-    }
-    output += '  }\n});';
-    return output;
+  /**
+   * Gets code-formatted load theme blob, specifically for the new theme designer,
+   * aka.ms/themedesigner. Can't use loadtheme like the old theme designer.
```

Comment:
Might be good to clarify why the new theme designer can't use loadTheme (I guessed at the explanation in the suggestion).
```suggestion
   * aka.ms/themedesigner. Shouldn't use loadTheme like the old theme designer since it's deprecated.
```